ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResourceTest.java

```diff
@@ -133,14 +135,18 @@
   @Mock
   private KsqlAuthorizationValidator authorizationValidator;
   private StreamedQueryResource testResource;
-  private PreparedStatement<Statement> statement;
+  private PreparedStatement<Statement> pushQueryStatement;
```

Comment:
Can we revert this change please? It's confusing to have `pushQueryStatement` used when testing PRINT TOPIC!

```
pushQueryStatement = PreparedStatement.of("print", cmd);
```

The test doesn't need specific fields for specific statement types.